x-pack/plugins/security_solution/common/endpoint/data_generators/exceptions_list_item_generator.ts

```diff
@@ -36,27 +36,18 @@ type UpdateExceptionListItemSchemaWithNonNullProps = NonNullableTypeProperties<
 
 export class ExceptionsListItemGenerator extends BaseDataGenerator<ExceptionListItemSchema> {
   generate(overrides: Partial<ExceptionListItemSchema> = {}): ExceptionListItemSchema {
-    return {
+    const exceptionItem: ExceptionListItemSchema = {
       _version: this.randomString(5),
       comments: [],
       created_at: this.randomPastDate(),
       created_by: this.randomUser(),
       description: 'created by ExceptionListItemGenerator',
       entries: [
         {
-          field: ConditionEntryField.HASH,
+          field: 'process.hash.md5',
```

Comment:
I noticed this while attempting to use it for FTR tests

Reply:
🔥